addons/tooltips/addon.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "Tooltips",
+  "description": "Adds tooltips for projects, users, studios, forum topics, and forum posts. Uses the Scratch API and ScratchDB.",
+  "credits": [
+    {
+      "name": "Hans5958"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "script.js",
+      "matches": "*"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "base.css",
+      "matches": "*"
+    },
+    {
+      "url": "scratch2.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "scratch2"
+        }
+      }
+    },
+    {
+      "url": "scratch3.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "scratch3"
+        }
+      }
+    },
+    {
+      "url": "extended.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "extended"
+        }
+      }
+    },
+    {
+      "url": "extended-statistics.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "extendedStatistics"
+        }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "id": "theme",
+      "name": "Theme",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "scratch3",
+          "name": "Scratch 3"
+        },
+        {
+          "id": "scratch2",
+          "name": "Scratch 2"
+        },
+        {
+          "id": "extended",
+          "name": "Extended"
+        },
+        {
+          "id": "extendedStatistics",
+          "name": "Extended (statistics)"
+        }
+      ],
+      "default": "scratch3"
+    },
+    {
+      "id": "color-scheme",
+      "name": "Color scheme",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "auto",
+          "name": "Auto"
+        },
+        {
+          "id": "light",
+          "name": "Light"
+        },
+        {
+          "id": "dark",
+          "name": "Dark"
+        }
+      ],
+      "default": "auto"
+    },
+    {
+      "id": "forceAll",
+      "name": "Force tooltips on all possible links",
+      "type": "boolean",
+      "default": false
+    }
+  ],
+  "tags": ["community", "projectPage", "profiles", "forums", "studios", "comments"],
```

Comment:
```suggestion
  "tags": ["community"],
```

Reply:
> not really because this should apply to more than just that
> use `community `only

Probably have a chat with @lisa-wolfgang because damn I'm still not sure.

Reply:
Confirm actions only has community and recommended

Reply:
I'd go with `["community", "featured"]`